docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[deploying-and-scaling]]
+== Deploying and Scaling Logstash
+
+The requirements for your Logstash installation change according to the scale demanded by your use case. In this 
```

Comment:
Maybe, "How you deploy and scale Logstash depends on your use case." and follow up with a couple contrasting examples?